src/runCommand.ts

```diff
@@ -66,7 +76,7 @@ async function runCommand(command, args) {
       } else {
         errorMessage = `Exited with code '${errorCode}'`;
       }
-      const improvedError = new Error(errorMessage);
+      const improvedError: Error & { cause?: string } = new Error(errorMessage);
```

Comment:
`cause` should be an error:
```suggestion
      const improvedError: Error & { cause?: Error } = new Error(errorMessage);
```